src/framework/auth/services/token/token.ts

```diff
@@ -14,6 +14,14 @@ export abstract class NbAuthToken {
   }
 }
 
+export class InvalidJWTTokenError extends Error {
+  constructor(message: string) {
+    super(message);
+    const actualPrototype = new.target.prototype;
```

Comment:
no need in additional variable here, just `Object.setPrototypeOf(this, new.target.prototype);`